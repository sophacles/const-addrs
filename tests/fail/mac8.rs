use const_addrs::mac8;

fn main() {
    let _ = mac8!("ca:fe:ca:fe:ca:fe");
}
