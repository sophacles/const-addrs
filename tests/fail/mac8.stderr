error: Invalid length of 17 characters
 --> tests/fail/mac8.rs:4:19
  |
4 |     let _ = mac8!("ca:fe:ca:fe:ca:fe");
  |                   ^^^^^^^^^^^^^^^^^^^
